src/inapp/inapp.ts

```diff
@@ -574,7 +569,7 @@ export function getInAppMessages(
               However there are 3 conditions in which to not show a message:
               
               1. _read_ key is truthy
-              2. _trigger.type_ key is "never"
+              2. _trigger.type_ key is "never" (deliver silently is checked)
```

Comment:
QoL comment... took me a while to figure out that this is the property that refers to silent delivery...